contracts/linear/src/validator_pool.rs

```diff
@@ -926,9 +930,15 @@ impl Validator {
         )
     }
 
-    pub fn on_stake_success(&mut self, pool: &mut ValidatorPool, amount: Balance) {
-        self.post_execution(pool);
-
+    pub fn on_stake_success(
+        &mut self,
+        pool: &mut ValidatorPool,
+        amount: Balance,
+        release_lock: bool,
```

Comment:
It seems `release_lock` is always `false` when calling this function from `validator_staked_callback()`?